bin/rrulebench.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+include __DIR__ . '/../vendor/autoload.php';
+
+if ($argc < 4) {
+    echo "sabre/vobject ", Sabre\VObject\Version::VERSION, " RRULE benchmark\n";
+    echo "\n";
+    echo "This script can be used to measure the speed of the 'recurrence expansion'\n";
+    echo "system.";
+    echo "\n";
+    echo "Usage: " . $argv[0] . " inputfile.ics startdate enddate\n";
+    die();
+}
+
+list(, $inputFile, $startDate, $endDate) = $argv;
+
+$bench = new Hoa\Bench\Bench();
```

Comment:
I had that in an earlier commit as `require-dev`, but it would break the build as we still support PHP 5.3. I can add it back in on the master branch, right after a merge.